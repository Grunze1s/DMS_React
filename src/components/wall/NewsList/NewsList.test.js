import React from 'react';
import NewsList from './index';
import {
    configure,
    shallow
} from 'enzyme';
import EnzymeAdapter from 'enzyme-adapter-react-16';
// JestHook.mock('expo-font');
configure({
    adapter: new EnzymeAdapter
})

const dummy_data={
    newsList: [{id:3},{id:1},{id:2}]
}
const setup = (props = {}, state = null) => {
    return (shallow( < NewsList {...props}  />)
    )
}

describe('<NewsList />', () => {
  

it("NewsList render", () => { 
        const wrapper= setup({...dummy_data});
        expect(wrapper).toMatchSnapshot();
  
})
})

