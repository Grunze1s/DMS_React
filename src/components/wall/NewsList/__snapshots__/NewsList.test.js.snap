// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NewsList /> NewsList render 1`] = `ShallowWrapper {}`;
